website/source/docs/providers/azurerm/r/virtual_network.html.markdown

```diff
@@ -57,7 +57,7 @@ The following arguments are supported:
 * `dns_servers` - (Optional) List of names of DNS servers previously registered
     on Azure.
 
-* `subnet` - (Required) Can be specified multiple times to define multiple
+* `subnet` - (Optional) Can be specified multiple times to define multiple
```

Comment:
Is it necessary here to use either all inline subnets or all separate resources? If so we should document this here _and_ in the subnet doc.